akumulid/server.h

```diff
@@ -70,14 +70,14 @@ struct ServerFactory {
 
     typedef std::function<std::shared_ptr<Server>(std::shared_ptr<AkumuliConnection>,
                                                   std::shared_ptr<ReadOperationBuilder>,
-                                                  const ServerSettings&)>
+                                                  ServerSettings&)>
```

Comment:
Yep, this is exactly the point. This object is used in many places so it's a good idea to keep it immutable.